isisdaeApp/Db/dae_diag.db

```diff
@@ -23,6 +23,32 @@ record(bi, "$(P)$(Q)DIAG:ENABLED")
     field(SCAN, "I/O Intr")
 }
 
+## Enable the diagnostics for a time period in seconds.
+##  This record will slowly count down until it gets to 0
+record(calc, "$(P)$(Q)KEEP:DIAG:ENABLE:FOR")
```

Comment:
As discussed, I think it would be better if the PV's tried to follow the standard naming convention. The subsystem in this case seems like it should be `DIAG:ENABLE:FOR` rather than `KEEP:DIAG:ENABLE:FOR`